wds-required-plugins.php

```diff
@@ -388,8 +388,9 @@ public function required_text_markup() {
 	 *
 	 * @return array
 	 */
-	public function filter_plugin_links( $actions = [], $plugin ) {
-
+	public function filter_plugin_links( $actions, $plugin ) {
```

Comment:
I see your comment, but if we are filtering an array, why are you getting deprecated messages?

Reply:
Sorry for beeing that late.

The error log says: 
`PHP Deprecated:  Required parameter $plugin follows optional parameter $actions in /shared/httpd/figuren/htdocs/content/mu-plugins/wds-required-plugins/wds-required-plugins.php on line 395`

Propably it's a better idea to just be a little more strict about the type of `$actions` like so:
`filter_plugin_links( array $actions, $plugin )`